grammars/Cargo.toml

```diff
@@ -15,7 +15,7 @@ rust-version = "1.65"
 
 [dependencies]
 pest = { path = "../pest", version = "2.7.15" }
-pest_derive = { path = "../derive", version = "2.7.15" }
+pest_derive = { path = "../derive", version = "2.7.15", features = ["grammar-extras"] }
```

Comment:
Alternatively: Would it be better to create a separate `./grammars_with_extras` crate? That way, we can also validate that the ones in this crate require only vanilla pest.

If you'd like to do that, I wonder if it'd make sense to just disable the new grammar, and then put that new work in as a separate PR? Just to keep the diff down on this one.

Reply:
Roger, I'll move this.